### PR TITLE
Flow Bite for Front End Components

### PR DESCRIPTION
**Pull request by Clairev3**
- Installed TailwindCSS & Flow Bite
- Added a simple (and ugly) logo, header and card to start a pattern for components
- Added VSCode workspace since there are/will be multiple apps to run

### CODE REVIEW

File: ui/src/lib/header.svelte [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+		DropdownItem,
+		DropdownHeader,
+		DropdownDivider,
+	} from 'flowbite-svelte';
+</script>
+<Navbar let:hidden let:toggle color="red">
+	<NavBrand href="/">
+		<img src="salt-shaker.png" class="mr-3 h-6 sm:h-9" alt="Flowbite Logo" />
+		<span class="self-center whitespace-nowrap text-xl font-semibold dark:text-white"
+			>Salt Mines</span
```
Comment (Nexushunter) [inline ui/src/lib/header.svelte:17]: Are we intending to call the site Salt Mines? If not this should probably match up with the name of the site.

File: ui/package-lock.json [hunk 2]
```diff
@@ -0,0 +1,5892 @@
+{
```
Comment (Nexushunter) [inline ui/package-lock.json:1]: ❗ 😬 Ahhhh npm was use to pull deps

**Review by Nexushunter — COMMENTED** [review]
It looks like it is gonna be simple to build out new components and pages. There's a couple nits though

File: ui/package-lock.json (hunk 2 above)
Comment (Clairev3) [reply ui/package-lock.json:1]: What should be used?

File: ui/src/lib/header.svelte (hunk 1 above)
Comment (Clairev3) [reply ui/src/lib/header.svelte:17]: No clue what this is really being called. I figured Salt Mines could at least be the code name for the project.

Comment (Clairev3) [reply ui/src/lib/header.svelte:17]: I don't expect any of these styles to be kept.

Comment (Nexushunter) [reply ui/src/lib/header.svelte:17]: Sounds good to me! Let's get mining!

File: ui/package-lock.json (hunk 2 above)
Comment (Nexushunter) [reply ui/package-lock.json:1]: `yarn` it's basically npm but easier and cleaner.

The global install is  `npm install --global yarn`

Comment (Clairev3) [reply ui/package-lock.json:1]: I ran that yarn command and it change anything. What am I missing?

Comment (Nexushunter) [reply ui/package-lock.json:1]: You just gotta remove the packages-lock.json.

Comment (Clairev3) [reply ui/package-lock.json]: Ok, did that. Does it look right now?

**Review by Nexushunter — APPROVED** [review]

**Pull request merged**